uiowa/uiowa
# Classrooms - Room list view

Resolves https://github.com/uiowa/uiowa/issues/6098

# To Test
```
ddev composer install && ddev blt frontend && ddev blt ds --site=classrooms.uiowa.edu
```

- Room list should be available for placement within basic page layout builder
- Filters form #6098 should all be accounted except for furniture.
- Teasers should look similar to mockup.
- Programmed Classrooms can be added to the site, but do not show in this room list. (e.g. https://api.maui.uiowa.edu/maui/api/pub/registrar/courses/AstraRoomData/BLB/235)
- Unit taxonomy pages are linked from the full node and if content is filled out, looks okay with a grid of rooms.
- Building link on the full node should link to a 404 /buildings/{building_id} in preparation for the next slice.
- Scheduling regions should link to node/151 instead of the term page.
- If you enable the person type filter for SiteNow People, you should still see a dropdown of person types.
